spec/spec_helper.rb

```diff
@@ -39,58 +40,4 @@
     # `true` in RSpec 4.
     mocks.verify_partial_doubles = false
   end
-
```

Comment:
What is the point of removing this comment? This removal only makes potential customization of tests more complicated.

Reply:
@matkoniecz When I ran `rubocop --auto-correct --safe` it double commented the block, the looks of which I didn't like so I removed it. But you have a valid point, I restored it now and it's formatted better.